packages/swing-store/src/snapStore.js

```diff
@@ -39,7 +39,7 @@ import { buffer } from './util.js';
  *   loadSnapshot: (vatID: string) => AsyncIterableIterator<Uint8Array>,
  *   saveSnapshot: (vatID: string, snapPos: number, snapshotStream: AsyncIterable<Uint8Array>) => Promise<SnapshotResult>,
  *   deleteAllUnusedSnapshots: () => void,
- *   deleteVatSnapshots: (vatID: string) => void,
+ *   deleteVatSnapshots: (vatID: string, budget?: number) => { done: boolean, cleanups: number },
```

Comment:
Suggestion: rename "cleanups" for clarity/comprehensibility.
```suggestion
 *   deleteVatSnapshots: (vatID: string, budget?: number) => { done: boolean, count: number },
```

Reply:
I'll look into it.. that'll require matching in the kernel, in the next PR in this series